docs/gallery.md

```diff
@@ -0,0 +1,120 @@
+
+Below is a gallery showcasing examples of such outputs.
+
+
+:::{grid} 1 1 2 2
+```{card} Jupyter Book 2 Workshop Template
+:link: https://jupyter-book.github.io/workshop-template/
+
+A GitHub Template repository designed for use in Jupyter Book 2 and MyST workshops.
+```
+````{card}
+:link: https://jupyter-book.github.io/workshop-template/
+```{figure} https://jupyter-book.github.io/workshop-template/build/logo-6b4f41e0d08ae79501fbae21e42b171f.svg
+
+```
+````
+:::
+
+:::{grid} 1 1 2 2
+```{card} The Turing Way handbook to reproducible, ethical and collaborative data science.
+:link: https://book.the-turing-way.org/
+
+The Turing Way is an open science, open collaboration, and community-driven project. We involve and support a diverse community of contributors to make data science accessible, comprehensible and effective for everyone.
+```
+````{card}
+:link: https://book.the-turing-way.org/
+```{figure} https://github.com/the-turing-way/the-turing-way/blob/main/book/figures/theturingway-pathway.jpg
+
+The Turing Way project illustration by Scriberia. Zenodo. [@https://doi.org/10.5281/ZENODO.3332807]()
+```
+````
+:::
+
+:::{grid} 1 1 2 2
+```{card} Myst Official Documentation
+:link: https://mystmd.org/guide
+
+MyST is an ecosystem of open-source, community-driven tools designed to revolutionize scientific communication. Our powerful authoring framework supports blogs, online books, scientific papers, reports and journals articles.
+```
+
+````{card}
+:link: https://mystmd.org/guide
+```{iframe} https://cdn.curvenote.com/0198d368-fb53-7691-80d4-a6f7a2c88ca5/public/links-7fffe87911b1e26605748d6f347d29d6.mp4
+:align: center
+
+Live graphs can be embedded directly in your documentation or articles with computation backed by Jupyter or JupyterLite – running locally, on Binder, or directly in your browser.
+```
+````
+:::
+
+:::{grid} 1 1 2 2
+```{card} Introducing Classical Mechanics & Special Relativity
+:link: https://freekpols.github.io/Mechanica/
+
+This book provides an introduction for freshman students into the world of classical mechanics and special relativity theory
+```
+````{card}
+:link: https://freekpols.github.io/Mechanica/
+```{figure} https://freekpols.github.io/Mechanica/build/cover-f2651612bb78120d7ad964dc4d5f8d4b.gif
+``` 
+````
+:::
+
+:::{grid} 1 1 2 2
+```{card} Scientific Python
+:link: https://scientific-python.org/
+
+The scientific Python ecosystem is a loose federation of community-developed and -owned Python projects widely used in scientific research, technical computing, and data science. The scientific Python community of contributors and maintainers are employed by a variety of universities, research labs, and companies.
+```
+````{card}
+:link: https://scientific-python.org/
+```{figure} https://scientific-python.org/images/logo.svg
+:class: framed
+
+
+```
+````
+:::
```

Comment:
SP is not yet myst; this should not be an entry in the gallery. -- If you are thinking of "projects that are supporting/behind/ally with myst/JB2; then SP should totally be there, but it should not be an example of it yet.
```suggestion

```